src/index.js

```diff
@@ -301,7 +302,7 @@ class Kronos extends Component {
       this.toggle(false)
       if (this.props.onBlur) this.props.onBlur(e)
     }
-    if (this.state.input == this.state.datetime.format(this.format())) {
+    if (this.state.input == datetime.format(this.format())) {
```

Comment:
```suggestion
    if (this.state.input == this.datetime?.format(this.format())) {
```

Why not just null check?

Reply:
Null check will also work, but they already initialized the `datetime` variable at the top and did not use it so i thought this is the right place to use.